fixtures/workers-with-assets-and-service-bindings/workerD-with-named-entrypoint/src/index.ts

```diff
@@ -1,6 +1,17 @@
 import { RpcTarget, WorkerEntrypoint } from "cloudflare:workers";
 
 export class EntrypointD extends WorkerEntrypoint {
+	#honey = "Bees make honey in worker-d";
+	#honeyBee = "I am worker-d's honeyBee prop";
+
+	get honey() {
```

Comment:
I wasn't sure...but am I doing props the right way here?

Reply:
`public foo = "bar"` and `instance.foo` should also work, I think?

Reply:
Yeah, to mirror the non-WorkerEntrypoint examples, want to try converting `#honeyBee` and its getter into just `public honeyBee = "I am worker-d's honeyBee prop"` here and in Worker C? I _think_ it should work, but maybe I'm wrong.

Reply:
nope, doesn't seem to work. tried it with `wrangler dev` as well. I'm getting `The RPC receiver does not implement the method "honeyBee".`